src/makers/win32/appx.js

```diff
@@ -65,7 +65,10 @@ export default async (dir, appName, targetArch, forgeConfig, packageJSON) => { /
     opts.devCert = await createDefaultCertificate(opts.publisher, outPath);
   }
 
-  if (!opts.publisher.match(/^CN=/)) {
+  if (typeof opts.publisher.match !== 'function') {
```

Comment:
I think using duck typing would be more useful if we expected that there would be more than one object type as input that had a compatible `.match` method. In this case, I think type checking reads better.